book/website/community-handbook/coworking/coworking-organisation.md

```diff
@@ -0,0 +1,131 @@
+If you organise these calls for your project members on a mutually agreed time and with an agenda to work on specific tasks, you can expect most of them to show up.
+
+If you keep your call open for any of your community members join, you may end up having a different number of participants on different calls.
+This would also mean that for some calls, there won't be anyone available to co-work with you.
+
+It's useful to handle some sort of sign-up sheet or registration system to organisers know who to expect on the call.
+In _The Turing Way_, we use dedicated HackMDs that are updated before each call.
+
+**Frequency of these calls**
+
+Once you identify the purpose, your target audience and the size of your call, you can decide how often you would like to host your calls.
+If you work with the same group of people, you can host it as per your common availability and a mutually agreed time on a weekly or monthly basis.
+
+If you host it for a group of people who work on multiple projects, you might consider hosting it on the same day and time each week or each month.
+This will allow them to check their availability in advance and block these hours in their calendar.
+This will also reduce the extra work of coordinating schedules every time (though you will have to do this in the beginning).
+
+**Resource availability**
+
+Based on the resources available to you, you might have to fine-tune other aspects of your call.
+For example:
+- Shared calendar: If there is a calendar that are updated with the coworking calls schedule, people can subscribe to them and indicate their availability.
+- Communication platforms: software like Zoom can allow you to connect with many people at the same time. However, you will need a paid subscription (pro account) to control who joins your call (by activating waiting room to avoid Zoom bombing), manage breakout rooms (to allow multiple small groups and discussions) or host longer calls (>40 minutes).
+- Chat system: If you have a group Slack or Gitter channel, information can be exchanged everyone, including those who can not join the calls but can offer suggestions or carry related tasks asynchronously but in coordination with the group.
+This also allows people to keep each other informed of any last-minute changes such as new call joining link, correct notes, or cancellation of calls.
+
+## Logistics and tasks
+
+Like any event, coworking calls required planning, organising, hosting and making sure that they happen.
+As organisers of the event, you will be required to:
+
+**1. Choose the tools**
+
+The following aspects will require you to make right choices in regards of what tool you will use.
+
+- maintaining shared notes: few options are [Etherpad](https://etherpad.org/), [HackMD](https://hackmd.io/), [GitHub](https://github.com/), [Google doc](https://docs.google.com/document/)
```

Comment:
This leads to google doc whereas all other Google links go to wikipedia. Maybe for consistency, they should all be wikipedia links, also I don't know how meaningful that link is if you don't have a google account and end up in your docs.

Reply:
Good point.